modules/nw-aws-nlb-new-cluster.aodc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic--aws-network-load-balancer.adoc
+
+[id="nw-aws-nlb-new-cluster_{context}"]
+= Configuring an Ingress Controller Network Load Balancer on a new AWS cluster
+
+You can create an ingress controller backed by an AWS Network Load Balancer (NLB) on a new cluster.
+
+.Prerequisites
+
+* You must be installing an AWS cluster with network customizations, and you must have completed the Creating the installation configuration file procedure.
+//** `PlatformStatus` of the infrastructure resource must be AWS.
+
+.Procedure
+
+Create an Ingress Controller backed by an AWS NLB on a new cluster.
+
+. Create a file named `cluster-ingress-default-ingresscontroller.yaml` with the following contents:
++
+----
+ $ cat cluster-ingress-default-ingresscontroller.yaml
+ apiVersion: operator.openshift.io/v1
+ kind: IngressController
+ metadata:
+   creationTimestamp: null
+   name: default
+   namespace: openshift-ingress-operator
+ spec:
+   endpointPublishingStrategy:
+     loadBalancer:
+       scope: External
+       providerParameters:
+         type: AWS
+         aws:
+           type: NLB
+     type: LoadBalancerService
+ status:
+   availableReplicas: 0
+   domain: ""
+   selector: ""
+----
+
+////
+add:
+
+step 3: Replace the file name in steps 2-5 of [3] with the "cluster-ingress-default-ingresscontroller.yaml" file you created.
```

Comment:
Maybe I'm missing it somewhere, but I don't see [3]. [3] should be https://docs.openshift.com/container-platform/4.5/installing/installing_aws/installing-aws-network-customizations.html#modifying-nwoperator-config-startup_installing-aws-network-customizations.